1-mandatory/2-quotegenerator/quotes.js

```diff
@@ -490,3 +492,92 @@ const quotes = [
     author: "Zig Ziglar",
   },
 ];
+
+
+  const quoteP = document.getElementById("quote");
+  const authorP = document.getElementById("author");
+  const newQuoteBtn = document.getElementById("newQuoteBtn");
+  newQuoteBtn.addEventListener("click", () => {
+  });
+
+  function generateNewQuote() {
+    return quotes[Math.floor(Math.random() * quotes.length)];
+  }
+ 
+  function displayQuote() {
+    const randomQuote = generateNewQuote();
+    quoteP.innerText = `"${randomQuote.quote}`;
+    authorP.innerText = `-${randomQuote.author}`;
+  }
+  displayQuote();
+
+
+// function pickFromArray(choices) {
+//   return choices[Math.floor(Math.random() * choices.length)];
+// }
+
+const quoteToggle = document.getElementById("quote-toggle");
+const quoteDisplay = document.getElementById("quote-display");
+const authorDisplay = document.getElementById("author-display");
```

Comment:
Really good! just for better readability we could group these DOM elements at the top with the others together